tests/validations.test.js

```diff
@@ -116,4 +121,32 @@ describe('>>>>> Validator Functions Tests', () => {
       expect(isValidIntRange(valueToTest)).toBeFalsy()
     })
   })
+
+  describe('.isStringWithComma', () => {
+    it('validates strings containing at least one comma', () => {
```

Comment:
Can we split each one of this tests into different `it` clauses?  I prefer to have only one `expect` per test (although this tests are really simple and small 😅)

Reply:
Absolutely! Will post a new commit in a sec...